waspc/data/Generator/templates/sdk/tsconfig.json

```diff
@@ -20,7 +20,8 @@
     "esModuleInterop": true,
     "moduleResolution": "node",
     "outDir": "dist",
-    "allowJs": true
+    "allowJs": true,
+    "types": ["@testing-library/jest-dom"],
```

Comment:
There were type errors related to the unit test matchers e.g. `toBeInTheDocument` was reported as undefined.